templates/goquorum/docker-compose.yml

```diff
@@ -633,8 +633,6 @@ services:
 
 {% endif %}
 
-{% if not orchestrate %}
```

Comment:
@ggarri Is this alright? Am not sure why prom/grafana were pulled out

Reply:
we decided to minimize the number of services to spin off when you are running Orchestrate to minimize the required resources. IMO it is good to minimize the number of services, or at least let users choose, so the minimum resources are kept to the least so it ensures we don't get a bad user experience due to lack of resources.